core/packages/test/scripts/build-binary.sh

```diff
@@ -30,6 +30,18 @@ rebuild_cumulus(){
     popd
 }
 
+build_cumulus_from_source(){
+    cumulus_src=$root_dir/cumulus
+    if [ ! -d "$cumulus_src" ] ; then
+        git clone https://github.com/Snowfork/cumulus $cumulus_src
+    fi
+    pushd $cumulus_src
+    git fetch origin && git switch $cumulus_version
```

Comment:
Aren't these steps typically performed using either `git submodule init` or `git submodule update`? Especially seeing as the branch name should already be defined in `.gitmodules`?